verify/go-tools/verify-govet.sh

```diff
@@ -17,4 +17,4 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-go vet -v $(go list ./... | grep -v /vendor/)
+go vet -v $(go list ./... | grep -v -E '/vendor/|verify$')
```

Comment:
do we need this if we also have golangci-lint running vet?

Reply:
That sounds like sound reasoning... but wouldn't it also apply to gometalinter (which also ran vet/vetshadow)?

Let me investigate and send a follow up.